ci/jenkins/test-mc.sh

```diff
@@ -430,6 +430,8 @@ function run_multicluster_e2e {
     fi
     set +x
     set -e
+
+    tar -zcf antrea-test-logs.tar.gz antrea-multicluster-test-logs
```

Comment:
Why is this line needed?

Reply:
It is used to archive artifacts for the Jenkins job so that after multicluster e2e tests are run, test logs are retained for that build